lightspeed/modules/proc_troubleshooting-vscode.adoc

```diff
@@ -38,3 +34,9 @@ To resolve this error, log out of VS Code and log in again using your Red Hat ac
 +
 {LightspeedShortName} service could take multiple seconds per task to populate the code recommendations. If you are using a multitask prompt, the {LightspeedShortName} service takes a bit longer to populate the results. Do not move your cursor or press any key while the code recommendation is being generated. If you change the cursor location or press any key, Ansible VS Code extension cancels the request and the {LightspeedShortName} service does not process your request. In this scenario, you must get the cursor back to its original position and repopulate the results.  
```

Comment:
I would insert "The" at the beginning of this sentence, before Lightspeed, and also right before Ansible VS Code extension in the second-to-last sentence .